proto/mysqlctl.proto

```diff
@@ -59,4 +62,35 @@ service MysqlCtl {
 message BackupInfo {
   string name = 1;
   string directory = 2;
+
+  string keyspace = 3;
+  string shard = 4;
+
+  // The following fields will be extracted from the .Name field. If an error
+  // occurs during extraction/parsing, these fields may not be set, but
+  // VtctldServer.GetBackups will not fail.
+
+  topodata.TabletAlias tablet_alias = 5;
+  vttime.Time time = 6;
+
+  // The following fields are may or may not be currently set. Work is inflight
```

Comment:
It might be useful to create + link an issue for this. Up to you!

Reply:
https://github.com/vitessio/vitess/issues/8332